docs/source/contributing/roadmap.md

```diff
@@ -0,0 +1,54 @@
+# The repo2docker roadmap
+
+This roadmap collects "next steps" for the project. The goal is to
+communicate where the current priorities are and where the project is heading.
+It is not a list aimed at limiting contributions to what is listed here. If
+something is not listed here please do bring it up in a new issue or start
+work on it.
```

Comment:
Maybe we can make the "get started" a link to https://repo2docker.readthedocs.io/en/latest/contributing/contributing.html#process-for-making-a-code-contribution. I was using it as a short way of saying "just because it isn't listed here doesn't mean it wouldn't be welcome. Use the usual procedures of making a successful contribution to an open-source project to make it happen. Start with small contributions and build up your understanding of the project and give the project a chance to get to know you. $alltheusualcaveatsofopensource"

WDYT?